guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,5 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
```

Comment:
How did you come up with this list?

Reply:
https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/3.4/1.2_release_notes.md#contributors

In https://github.com/theforeman/foreman-documentation/issues/1767 I kept notes on how I want to do this in this repository.